node/runtime/src/lib.rs

```diff
@@ -58,8 +58,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 99,
-	impl_version: 104,
+	spec_version: 100,
+	impl_version: 105,
```

Comment:
Convention is to make `impl_version` the same if incrementing `spec_version`.
```suggestion
	impl_version: 100,
```